app/views/users/edit.html.erb

```diff
@@ -79,9 +79,18 @@ minlength: User.password_length.min %>
      <%= form.label :receive_email_notifications, t("communication_preferences.email_me") %>
    </div>
    <div>
-     <%= form.check_box :receive_sms_notifications, class: "toggle-sms-notifications" %>
+     <%= form.check_box :receive_sms_notifications, id: "toggle-sms-notifications", class: "toggle-sms-notifications" %>
```

Comment:
Some things are dependent on the automatically generated id  `user_receive_sms_notifications`. Don't think it's best to override this.